apps/website/vite.config.ts

```
@@ -1,18 +1,12 @@
 /// <reference types='vitest' />
+// Patch Node’s webcrypto into the browser global
+import { webcrypto } from 'crypto';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)